src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -1015,34 +1015,72 @@ namespace CoolingPanelSimple {
 		
 		RegisterPlantCompDesignFlow( CoolingPanel( CoolingPanelNum ).WaterInletNode, CoolingPanel( CoolingPanelNum ).WaterVolFlowRateMax );
 
+		bool SizeCoolingPanelUASuccess;
+		SizeCoolingPanelUASuccess = SizeCoolingPanelUA( CoolingPanelNum );
+		if ( ! SizeCoolingPanelUASuccess ) ShowFatalError( "SizeCoolingPanelUA: Program terminated for previous conditions." );
+
+	}
+
+	bool
+	SizeCoolingPanelUA(
+					 int const CoolingPanelNum
+					 )
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Rick Strand
+		//       DATE WRITTEN   June 2017
+		
+		// PURPOSE OF THIS SUBROUTINE:
+		// This subroutine sizes UA value for the simple chilled ceiling panel.
+
+		// Return value
+		bool SizeCoolingPanelUA;
+		
 		// These initializations are mainly the calculation of the UA value for the heat exchanger formulation of the simple cooling panel
+		Real64 Cp;
 		Real64 MDot;
 		Real64 MDotXCp;
 		Real64 Qrated;
 		Real64 Tinletr;
 		Real64 Tzoner;
+		Real64 RatCapToTheoMax; // Ratio of unit capacity to theoretical maximum output based on rated parameters
+
+		SizeCoolingPanelUA = true;
 		Cp = 4120.0; // Just an approximation, don't need to get an exact number
```

Comment:
Why don't we need an exact number if a few lines below we test for RatCapToTheMax < 1.0 ?? What if RatCapToTheMax is 1.0001 because we weren't accurate on Cp?